src/app/package.json

```diff
@@ -6,21 +6,25 @@
     "dependencies": {
         "@google/markerclusterer": "1.0.3",
         "@material-ui/core": "3.1.0",
+        "@hot-loader/react-dom": "16.8.6",
```

Comment:
Can this  be moved up a line to maintain alphabetical ordering?

Reply:
Oh yeah -- I originally did have it alphabetically but forgot to make it so again after rebasing on develop.

Reply:
I moved this in a new fixup commit then squashed it back into the original.